interfaces/seccomp/template.go

```diff
@@ -543,7 +543,23 @@ fdatasync
 fsync
 syncfs
 sysinfo
+
+# At one point, we relied on AppArmor denying CAP_SYSLOG to prevent privileged
+# syslog(2) access. However, as of Jammy, the kernel.dmesg_restrict sysctl has
+# been set to 1 by default, which requires CAP_SYSLOG for access to /dev/kmsg.
+# Thus, we have to grant CAP_SYSLOG to the log-observe interface, so we must
+# explicitly deny the non-"observe" type privileged accesses here. The following
+# only allows SYSLOG_ACTION_READ{_ALL}, SYSLOG_ACTION_SIZE_{UNREAD,BUFFER}
```

Comment:
I wonder whether we still need the "capability syslog" in kernel_module_observe.go ? cc @alexmurray

Reply:
Hmm this is a good point @pedronis - yes I think that with this new reworking of log_observe we can remove this `capability syslog` from kernel-module-observe and instead then just have snaps plug log-observe as well (if they don't already) for that purpose. Or I wonder also if we should have similar seccomp rules in place in kernel-module-observe as well to avoid granting more privileged access than is required?